src/Misc/layoutbin/en-US/strings.json

```diff
@@ -245,6 +246,13 @@
     "--username     basic authentication username",
     "--password     basic authentication password",
     "",
+    "",
+    "Arguments for enableautologon (Only on Windows):",
+    "-----------------------------------------------------------------------------",
+    "--restartnow          Used after autologon configuration is done. Recommended to restart the machine for the configured settings to take place.",
```

Comment:
Removed the prompt for screensaver disable action.

---
In reply to: [126192719](https://github.com/Microsoft/vsts-agent/pull/1063#discussion_r126192719) [](ancestors = 126192719,126191138)